pkgs/by-name/su/surfer/package.nix

```
@@ -0,0 +1,71 @@
+{
+  lib,
+  fetchFromGitLab,
+  rustPlatform,
+  pkg-config,
+  openssl,
+  wayland,
+  autoPatchelfHook,
+  libxkbcommon,
+  libGL,
+  libX11,
+  libXcursor,
+  libXi,
+  stdenv,
+}:
+rustPlatform.buildRustPackage rec {
+  pname = "surfer";
+  version = "0.2.0";
+
+  src = fetchFromGitLab {
+    owner = "surfer-project";
+    repo = "surfer";
+    rev = "v${version}";
+    hash = "sha256-C5jyWLs7fdEn2oW5BORZYazQwjXNxf8ketYFwlVkHpA";
+    fetchSubmodules = true;
+  };
+
+  nativeBuildInputs = [
+    pkg-config
+    autoPatchelfHook
+  ];
+
+  buildInputs = [
+    openssl
+    wayland
+    libxkbcommon
+    libGL
+    stdenv.cc.cc.libgcc
+  ];
+
+  runtimeDependencies = [
+    openssl
+    wayland
+    libxkbcommon
+    libGL
+    libX11
+    libXcursor
+    libXi
```

```suggestion
    libX11
    libXcursor
    libXi
```
are these not detected by autoPatchelfHook?

---

When I remove them and run surfer on Ubuntu using NixGL I get:
```
Message:  called `Result::unwrap()` on an `Err` value: WinitEventLoop(NotSupported(NotSupportedError))
```

It works when I add the libs back.

---

This is also required because of dlopen. See https://github.com/NixOS/nixpkgs/pull/324637#discussion_r1667592609. Maybe add a comment?